airflow/www_rbac/views.py

```diff
@@ -146,6 +146,29 @@ def render(self, template, **context):
 
 
 class Airflow(AirflowBaseView):
+    @expose('/health')
```

Comment:
do we need access control for this endpoint? I assume only admin should access this endpoint?

Reply:
Thanks @feng-tao  for the review!

The current implementation is not requiring any logging-in to access `/health` endpoint. But of course it only returns a very "safe" response "`The server is healthy!`" (which is not useful/accurate).

For this PR, I would still like to suggest not require admin or logging-in to access this endpoint. This is just like the status page of GitHub. Everyone should be able access it without any logging-in. But on the other hand, we do need to be careful of what to expose via this endpoint. This was partially covered in my last reply comment to Kaxil.